src/midi/fluid_seq.cpp

```diff
@@ -39,44 +57,251 @@
 
 #define FLUID_SEQUENCER_EVENTS_MAX	1000
 
+/* Private data for clients */
+typedef struct _fluid_sequencer_client_t
+{
+    fluid_seq_id_t id;
+    std::string name;
+    fluid_event_callback_t callback;
+    void *data;
+} fluid_sequencer_client_t;
+
 /* Private data for SEQUENCER */
 struct _fluid_sequencer_t
```

Comment:
prob want to add "final" to these `struct _fluid_sequencer_t final`

Reply:
Maybe, I'll think about it.